.github/workflows/helm-release.yaml

```
@@ -72,29 +87,72 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
 
-      - name: Set up python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "${{ inputs.python-version }}"
-
       - name: Bump version
         id: release-tag
-        uses: bakdata/ci-templates/actions/bump-version@v1.0.0
+        uses: bakdata/ci-templates/actions/bump-version@v1.5.2
         with:
           release-type: ${{ inputs.release-type }}
+          working-directory: ${{ inputs.charts-dir }}
 
-      - name: Commit and push bumpversion.cfg file
-        uses: bakdata/ci-templates/actions/commit-and-push@v1.0.0
+      - name: Download previous Helm index
+        run: curl -O -f ${{ inputs.page-url }}/index.yaml
+        if: ${{ inputs.skip-download == 'false' }}
+        shell: bash
+        working-directory: ${{ inputs.charts-dir }}
+
+      - name: Find previous package versions
+        id: packages
+        uses: mikefarah/yq@v4.28.1
+        if: ${{ inputs.skip-download == 'false' }}
+        with:
+          cmd: yq '.entries.[] | map(.urls.[]) | join(",")' ${{ inputs.charts-dir }}/index.yaml
+
+      - name: Create Helm package & update index
+        uses: bakdata/ci-templates/actions/helm-package@v1.6.0
+        with:
+          charts-dir: ${{ inputs.charts-dir }}
+
+      - name: Commit and push Chart.yaml & .bumpversion.cfg file
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.6.0
         with:
           ref: ${{ inputs.ref }}
           commit-message: "Bump version ${{ steps.release-tag.outputs.old-tag }} → ${{ steps.release-tag.outputs.release-tag }}"
           github-username: ${{ secrets.github-username }}
           github-email: ${{ secrets.github-email }}
           github-token: ${{ secrets.github-token }}
 
-      - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.2.1
+      - name: Prepare artifact
+        run: |
+          mkdir ${{ inputs.artifact-dir }}
+          cp index.yaml ${{ inputs.artifact-dir }}/
+          find . -maxdepth 1 -type f -name "*.tgz" -exec mv {} ${{ inputs.artifact-dir }} \;
+        shell: bash
+        working-directory: ${{ inputs.charts-dir }}
+
+      - name: Download packages
+        run: curl -O -f "${{ inputs.page-url }}/{${{ steps.packages.outputs.result }}}"
```

What is the reason for the third pair of curly braces here?
```suggestion
        run: curl -O -f "${{ inputs.page-url }}/${{ steps.packages.outputs.result }}"
```